src/listpack.c

```diff
@@ -482,19 +537,23 @@ unsigned char *lpNext(unsigned char *lp, unsigned char *p) {
 /* If 'p' points to an element of the listpack, calling lpPrev() will return
  * the pointer to the previous element (the one on the left), or NULL if 'p'
  * already pointed to the first element of the listpack. */
+/* 如果 'p' 指向 listpack 中的一个元素，
+ * 调用 lpPrev() 将返回该指针的上一个元素（左边的那个），
+ * 或返回 NULL 如果 'p' 已经指向 listpack 的第一个元素。*/
 unsigned char *lpPrev(unsigned char *lp, unsigned char *p) {
     assert(p);
     if (p-lp == LP_HDR_SIZE) return NULL;
-    p--; /* Seek the first backlen byte of the last element. */
+    p--; /* Seek the first backlen byte of the last element. */ /* 找到最后一个元素的第一个 backlen 字节。*/
```

Comment:
```suggestion
    p--; /* Seek the first backlen byte of the last element. */
```
既然566行把英文和中文注释分行写了这里也起新的一行写中文注释吧，还有这句应该说“找到最后一个元素的backlen的第一个字节"吧，而不是"第一个backlen字节"这样有点怪

Reply:
Sorry，上次没检查到位